pkg/volume/empty_dir/empty_dir.go

```diff
@@ -206,7 +207,7 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 	// the container.
 	opts := ed.getTmpfsMountOptions()
 	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
-	return ed.mounter.Mount("tmpfs", dir, "tmpfs", 0, opts)
+	return ed.mounter.Mount("tmpfs", dir, "tmpfs", nil)
```

Comment:
Yes, I also noticed this.  I believe it's a merge problem.  Thanks for the heads up!